### PR TITLE
6.2.4.final  bootable jar

### PR DESCRIPTION
**Pull request by tommaso-borgato**
This PR is to be changed because it has to against some branch derived from the 6.2.4.Final tag (e.g. 6.2.4.SPx)

This PR:
-  reintroduces bootable jar tests
- adds option to run bootable jar tests with a channel manifest

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -53,6 +53,7 @@
         <version.org.jboss.galleon>5.1.0.Final</version.org.jboss.galleon>
         <version.org.wildfly.galleon-plugins>6.4.2.Final</version.org.wildfly.galleon-plugins>
         <version.org.wildfly.plugins.wildfly-maven-plugin>4.0.0.Final</version.org.wildfly.plugins.wildfly-maven-plugin>
```
Comment (fabiobrz) [inline pom.xml:55]: I guess this should be bumped as well.

File: pom.xml [hunk 2]
```diff
@@ -53,6 +53,7 @@
         <version.org.jboss.galleon>5.1.0.Final</version.org.jboss.galleon>
         <version.org.wildfly.galleon-plugins>6.4.2.Final</version.org.wildfly.galleon-plugins>
```
Comment (fabiobrz) [inline pom.xml:54]: And maybe these Galleons dependency will need to be updated according to those of the wildfly-maven-plugin and wildfly-jar-maven-plugin? Not sure though, but I think this should be worth checking.

**Review by fabiobrz — CHANGES_REQUESTED** [review]
Hi @tommaso-borgato - I went through the changes and dropped a couple of minor comments.
BTW - Looking at the original [PR that introduced the changes originally](https://github.com/resteasy/resteasy/pull/2558) I noticed you've not documented the changes in the README, and IIUC also the filtered tests seem to miss some comments about the reason for the exclusion. WDYT?

**Comment by tommaso-borgato** [comment]
We are going to use https://github.com/tommaso-borgato/Resteasy/tree/6.2.4.Final--bootable-jar

**Comment by tommaso-borgato** [comment]
We are going to use https://github.com/tommaso-borgato/Resteasy/tree/6.2.4.Final--bootable-jar

**Pull request closed without merging**